collectors/o365/healthcheck.js

```diff
@@ -12,21 +12,22 @@
 
 const o365_mgmnt = require('./lib/o365_mgmnt');
 const al_health = require('@alertlogic/al-aws-collector-js').Health
-
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
 /*
  * Checks the subscriptions against the configured content type. Starts the subscriptions if th
  */
 
 function checkO365Subscriptions(callback){
-    return o365_mgmnt.listSubscriptions()
+    return PawsCollector.load().then(({pawsCreds}) => {
```

Comment:
according to this [code](https://github.com/alertlogic/al-azure-collector-js/blob/master/master.js#L309) first arg is collector wher eyou can probably get `pawsCreds` from.
I think we need to just bind health check function context so that we can use `this` inside it

Reply:
Context binding already done [here](https://github.com/alertlogic/al-aws-collector-js/blob/a22017e9e6033d16f8fcf7596039f79afc0d010a/al_aws_collector.js#L468). So you should be able to use `this.pawsCreds` inside this function

Reply:
sure. I will check and update.